inflammation/models.py

```diff
@@ -1,108 +1,32 @@
-        }
-    ]
-
-    :param filename: Filename of CSV to load
-
-    """
-    with open(filename, 'r', encoding='utf-8') as file:
-        data_as_json = json.load(file)
-        return [np.array(entry['observations']) for entry in data_as_json]
-
 def daily_mean(data):
-    """Calculate the daily mean of a 2D inflammation data array.
-
-    :param data: A 2D data array with inflammation data
-    (one row represents one patient and one col one day)
-    :returns: An array of mean values of measurements for each day
-    """
+    """Calculate the daily mean of a 2d inflammation data array."""
     return np.mean(data, axis=0)
-
 def daily_max(data):
-    """Calculate the daily max of a 2D inflammation data array.
-
-    :param data: A 2D data array with inflammation data
-    (one row represents one patient and one col one day)
-    :returns: An array of max values of measurements for each day
-    """
+    """Calculate the daily max of a 2d inflammation data array."""
     return np.max(data, axis=0)
-
 def daily_min(data):
-    """Calculate the daily min of a 2D inflammation data array.
-
-    :param data: A 2D data array with inflammation data
-    (one row represents one patient and one col one day)
-    :returns: An array of min values of measurements for each day
-    """
+    """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
-def patient_normalise(data):
-    """
-    Normalise patient data between 0 and 1 of a 2D inflammation data array.
-
-    Any NaN values are ignored, and normalised to 0
-
-    :param data: 2D array of inflammation data
-    :type data: ndarray
-
-    """
-    if not isinstance(data, np.ndarray):
-        raise TypeError('data input should be ndarray')
-    if len(data.shape) != 2:
-        raise ValueError('inflammation array should be 2-dimensional')
-    if np.any(data < 0):
-        raise ValueError('inflammation values should be non-negative')
-    max_data = np.nanmax(data, axis=1)
-    with np.errstate(invalid='ignore', divide='ignore'):
-        normalised = data / max_data[:, np.newaxis]
-    normalised[np.isnan(normalised)] = 0
-    return normalised
-
-def compute_standard_deviation_by_day(data):
-    """
-    Calculates the standard deviation for a set of daily inflammation means
-    """
-    means_by_day = map(daily_mean, data)
-    means_by_day_matrix = np.stack(list(means_by_day))
-
-    daily_standard_deviation = np.std(means_by_day_matrix, axis=0)
-    return daily_standard_deviation
-
-def analyse_data(data_source):
-    """Calculate the standard deviation by day between datasets
-    Gets all the inflammation csvs within a directory, works out the mean
-    inflammation value for each day across all datasets, then graphs the
-    standard deviation of these means."""
-    data = data_source.load_inflammation_data()
-    daily_standard_deviation = compute_standard_deviation_by_day(data)
 
-    return daily_standard_deviation
+def s_dev(data):
```

Comment:
Other function names are in a structure like; "daily_mean, daily_min, daily_....etc.". I would recommend changing this function name to one of similar structure.

Reply:
True. Addressed in commit 86db192.